app/controllers/subjects_controller.rb

```diff
@@ -7,8 +7,29 @@ def index
   
   def show
   end
+
+  def new
+    @subject = Subject.new
+  end
+
+  def create
+    @subject = Subject.new(subject_params)
+    respond_to do |format|
+      if @subject.save
+        format.html { redirect_to @subject, notice: 'Subject was successfully created.' }
+        format.json { render :show, status: :created, location: @subject }
+      else
+        format.html { render :new }
+        format.json { render json: @s.errors, status: :unprocessable_entity }
```

Comment:
TBH i copied and pasted those lines from the posts controller - but i guess its probably wrong? is there another `.format` i need to use?